### ISSUE TITLE
settings not save 

### ISSUE THREAD

**Issue by koko009** [jellyfin-adult/Jellyfin.Plugin.PhoenixAdultRebirth#40]
hey 
plug is working but i have a proplem 
i want to use api metadata metadataapi.net 
 but when change any thing in setting then tick on save button 
page refersh auto and not save any thing 

edit: 2
[Porn Fidelity] site not working on search
wtfpass           site not working
wowgirls         site not working 
and not supported 

wahen search example 
data18 - 1210649        not working 
data18.com - 1210649    not working 

i hope solve this proplem via api meteadata.net 
wahen iam use plex with plug the porndb and phonix adult all this is good and working fine without change any name files 

emby 4.8
plusg latset vr

**Comment by jellyfin-adult** [comment]
I've found the root cause, it looks like it was never working because the current settings page implementation only considers `Jellyfin` not the `Emby`. I'll try to fix it ASAP

**Comment by koko009** [comment]
> I've found the root cause, it looks like it was never working because the current settings page implementation only considers `Jellyfin` not the `Emby`. I'll try to fix it ASAP

Thanks man 
Can you explain to me how to search for files through the sites you mentioned, because they do not work for me and I cannot download file data, especially from the site  data18.com

**Comment by jellyfin-adult** [comment]
@koko009 In general the rule is quite simple, for a plugin to be able to get some metadata you need to follow these format:

```
siteName - yyyy-MM-dd - [optional ID] title.mp4/mkv
```

For example for data18

```
data18 - 2024-03-02 - The Tickle Monster.mp4
data18 - 2021-04-17 - Cuma Sutra Magick.mkv
```

Sometimes, a search without a date will work as well (but especially on `data18` it can be helpful because there are a lot of videos with the same name.

**Comment by jellyfin-adult** [comment]
Fixed in 2.0.4.67
